backends/qualcomm/serialization/qc_compiler_spec.fbs

```diff
@@ -32,16 +32,17 @@ table HtpInfo {
 /// to get SoC Model in supported snapdragon devices
 enum QcomChipset: int {
   UNKNOWN_SM = 0,
+  SA8295 = 39,
```

Comment:
Looks good, though it's another BC breaking change. Is there a way to make it BC?